Client/core/CChat.cpp

```diff
@@ -530,8 +546,10 @@ void CChat::ClearInput ( void )
     m_strInputText.clear ();
     m_InputLine.Clear ();
     m_vecInputSize = CalcInputSize ();
-    if ( m_pInput )
+    if ( m_pInput ) {

```

Comment:
Our (unwritten) styleguide says that braces should be placed in the next line (there are a couple of other occurrences in the PR). For example:
```cpp
// CORRECT
if (m_pInput)
{
    // ...
}
```